library/build.gradle

```diff
@@ -41,25 +40,11 @@ dependencies {
     implementation group: 'com.azure.resourcemanager', name: 'azure-resourcemanager-applicationinsights', version: '1.0.0'
     implementation group: 'com.azure.resourcemanager', name: 'azure-resourcemanager-securityinsights', version: '1.0.0-beta.4'
 
-    implementation group: "io.kubernetes", name: "client-java", version: "20.0.0-legacy"
-    // io.kubernetes:client-java brings in snakeyaml 2.0 which breaks whatever version of liquibase was pulled in before
-    implementation group: 'org.liquibase', name: 'liquibase-core', version: '4.26.0'
```

Comment:
There is no longer an incompatibility, from what I could see. This set of upgrades gets us snakeyaml 2.2 across the board.